src/frontend/Felles/HeaderMedSøk/HeaderMedSøk.tsx

```diff
@@ -71,6 +71,12 @@ const lagEksterneLenker = (
             name: 'Uttrekk arbeidssøkere (P43)',
             href: '/uttrekk/arbeidssoker',
         });
+        if (toggles[ToggleName.opprettBehandlingForFerdigstiltJournalpost]) {
+            eksterneLenker.push({
+                name: 'Opprett førstegangsbehandling manuelt',
+                href: '/opprett-forstegangsbehandling',
+            });
+        }
```

Comment:
Spørsmål: Skal dette være tilgjengelig for alle saksbehandlere, eller bare for coacher/admins? (på samme måte som opprettBehandlingForFerdigstiltJournalpost)

Reply:
Usikker - tenker Lars/Mirja kan vurdere det. Foreløpig er det togglet, men tror det